<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="Nikhil" failures="1" tests="1" name="FICTION_Test" time="115.648" errors="0" timestamp="2023-02-24T10:49:44 IST">
  <testcase classname="Test_Class_Gutenberg.CGF_0005_FICTION" name="CGF_0005Fiction" time="45.172">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	expected [Fiction] but found [FICTION]">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
expected [Fiction] but found [FICTION]
at Test_Class_Gutenberg.CGF_0005_FICTION.CGF_0005Fiction(CGF_0005_FICTION.java:34)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
... Removed 17 stack frames]]>
    </failure>
  </testcase> <!-- CGF_0005Fiction -->
</testsuite> <!-- FICTION_Test -->
